app/assets/stylesheets/global/_base.scss

```diff
@@ -67,3 +97,80 @@ input[type="submit"] {
   margin-left: 44%;
   margin-right: 10%;
 }
+
+.skip-header {
+  width: 100%;
+  a {
+    width: calc(100% - 0.5rem);
+    height: 0;
+    padding: 0;
+    display: block;
+    box-sizing: border-box;
+    &:focus {
+      height: max-content;
+      padding: 0.25rem;
+      margin: 0.5rem;
+    }
+  }
+}
+
+.banner {
+  background-color: $color_bg-dark;
+  position: relative;
+  padding-top: 2em;
+  padding-bottom: 8rem;
+  color: $color_font-light;
+  font-size: 20px;
+  font-weight: normal;
+  background: $color_bg-dark;
+  background-size: 40%;
+
+  .center {
+    margin: 8rem auto 0 auto;
+    width: fit-content;
+  }
+
+  h1 {
```

Comment:
Below this point in the file looks like a merge conflict issue. These headers shouldn't be redefined at the bottom of the page.

Reply:
The reason for this approach can be seen by reviewing the "search" page where the default refinery search result text is output in an H1 on the search page, which sort of defies our convention of placing the h1 inside the banner (which is how our mocks tell us to do it). I am open to constructive suggestions of an alternative approach for this, but recommend reviewing a page with search results before doing so. Otherwise I'm going to stand by this approach since I do not have any other bullets in the chamber for resolving this particular challenge.